README.md

```diff
@@ -120,9 +120,9 @@ Data which is stored upon startup includes:
 
 * It is easiest to use the development virtual machine to work on this project. Follow [these instructions](RemoteDesktopToVM.md) to set up and connect to the virtual machine using your Gordon account.
 
-* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash.
+* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash. It is possible that you will need to add the SSH key to your Git account, instructions to generate one can be found [here](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh). You will then need to `cd ssh` then `cat id_rsa.pub`, copy the output and go to your github settings and paste it in your SSH keys.
 
-  * Before you open the gordon-360-api folder, you will have to add the `secrets.config` file to it. The file is located on the CS-RDSH-02 virtual machine in `C:\Users\Public\Public Documents\` (or `/c/users/public/documents\` when in git-bash). Copy the file `secrets.config` to the same folder in your project that contains the `web.config` file; currently, this is in `gordon-360-api\Gordon360`. This file is a sort of keyring for the server to authorize itself at various points.
+  * Before you open the gordon-360-api folder, you will have to add the `secrets.config` file to it. The file is located on the CS-RDSH-02 virtual machine in `C:\Users\Public\Public Documents\` (or `/c/users/public/documents\` when in git-bash). Copy the file `secrets.config` to the same folder in your project that contains the `web.config` file; currently, this is in `gordon-360-api\Gordon360`. This file is a sort of keyring for the server to authorize itself at various points. You can do this by CDing into the gordon360 folder and typing `cp /C/users/public/documents/secrets.config .`
```

Comment:
Although it technically doesn't matter on Windows, the folder "Gordon360" in the gordon-360-api respository should be capitalized as that's how it appears on case-sensitive systems like Github and linux.